<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculConvertor</class>
 <widget class="QMainWindow" name="CalculConvertor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalculConvertor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="SValue">
        <property name="text">
         <string>Result</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="SInput">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Choose parametrs:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QRadioButton" name="SDec">
           <property name="text">
            <string>Decimal</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="SBin">
           <property name="text">
            <string>Binary</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="SHex">
           <property name="text">
            <string>Hexadecimal</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_1">
      <item>
       <widget class="QLabel" name="FValue">
        <property name="text">
         <string>Value</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="FInput"/>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Choose parametrs:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QRadioButton" name="FDec">
           <property name="text">
            <string>Decimal</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="FBin">
           <property name="text">
            <string>Binary</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="FHex">
           <property name="text">
            <string>Hexadecimal</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="SuccessLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="ButtonConvert">
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="Dark">
      <property name="text">
       <string>Dark</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="Light">
      <property name="text">
       <string>Light</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionSelectFont"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSelectFont">
   <property name="text">
    <string>Select Font</string>
   </property>
   <property name="toolTip">
    <string>Select the display font</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
